pcsx2/DEV9/InternalServers/DNS_Logger.h

```diff
@@ -30,6 +30,9 @@ namespace InternalServers
 		void InspectSend(PacketReader::IP::IP_Payload* payload);
 
 	private:
+		std::string VectorToString(std::vector<u8> data);
```

Comment:
If you pass the vector by const reference, you won't make a copy of it., e.g. `const std::vector<u8>& data`

Reply:
I always forget about reference (and apparently also forgot to make it const)